client/my-sites/stats/activity-log-banner/success-banner.jsx

```diff
@@ -46,15 +49,19 @@ class SuccessBanner extends PureComponent {
 		downloadCount: PropTypes.number,
 
 		// connect
-		dismissRewindRestoreProgress: PropTypes.func.isRequired,
+		dismissRestoreProgress: PropTypes.func.isRequired,
+		dismissBackupProgress: PropTypes.func.isRequired,
 		recordTracksEvent: PropTypes.func.isRequired,
 
 		// localize
 		moment: PropTypes.func.isRequired,
 		translate: PropTypes.func.isRequired,
 	};
 
-	handleDismiss = () => this.props.dismissRewindRestoreProgress( this.props.siteId );
+	handleDismiss = () =>
+		this.props.siteUrl
```

Comment:
can we document how this property is indicating whether or not we have a backup or a restore? the relationship is quite blurry

Reply:
Good catch. I fixed it. It was supposed to be `backupUrl`.